src/_imagingft.c

```diff
@@ -498,20 +506,20 @@ text_layout_raqm(PyObject* string, FontObject* self, const char* dir, PyObject *
         goto failed;
     }
 
-    if (p_raqm.version == 1){
-        for (i = 0; i < count; i++) {
+    for (i = 0; i < count; i++) {
+        if (p_raqm.version == 1) {
```

Comment:
I think it would be faster to check the condition outside the loop.

Reply:
You're right. I've updated the commit

Reply:
I'm having trouble finding a small CJK font that could be included within the Pillow license. Do you have any thoughts?